src/main/webapp/WEB-INF/jsp/signup.jsp

```diff
@@ -25,8 +23,52 @@ label {
 }
 </style>
 
+<script type="text/javascript">
+
+   //Created / Generates the captcha function    
+    function DrawCaptcha()
+    {
+    	document.getElementById("submitbutton").disabled = true;
+        var a = Math.ceil(Math.random() * 10)+ '';
+        var b = Math.ceil(Math.random() * 10)+ '';       
+        var c = Math.ceil(Math.random() * 10)+ '';  
+        var d = Math.ceil(Math.random() * 10)+ '';  
+        var e = Math.ceil(Math.random() * 10)+ '';  
+        var f = Math.ceil(Math.random() * 10)+ '';  
+        var g = Math.ceil(Math.random() * 10)+ '';  
+        var code = a + ' ' + b + ' ' + ' ' + c + ' ' + d + ' ' + e + ' '+ f + ' ' + g;
+        document.getElementById("txtCaptcha").value = code;
+        document.getElementById("txtInput").value = '';
+    }
+
+    // Validate the Entered input aganist the generated security code function   
+    	
+	function ValidCaptcha(){

```

Comment:
Also I did not do any TransactionController changes today.... It might be a mistake,,,,I am nt even seeing any conficlts with master now...